.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     container: qmkfm/qmk_cli
 
     steps:
-    - uses: actions/checkout@v2
```

Comment:
why's this changed in a keyboard pr?

Reply:
Sorry, my fault. Can you please remove the change in ci.yml?